scalableminds/webknossos
# Short links

### URL of deployed dev instance (used for testing):
- https://featureshortlinks.webknossos.xyz

### Steps to test (Backend):
- Create a shortlink (POST /api/shortLinks)
- Read a shortlink (GET /api/shortLinks/byKey/:key)
- Check sensible errors when giving invalid parameters

### Steps to test (end-to-end)
- In Annotation sharing modal, copy short link
- In new tab, redirect should work

### Issues:
- fixes part of #6309 
- This is just allows for the creation of links, routing shortlinks and converting them is not implemented

------
(Please delete unneeded items, merge only when none are left open)
